package.json

```diff
@@ -9,14 +9,17 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "express": "^4.18.2",
+    "@types/mongoose": "^5.11.97",
     "@types/express": "^4.17.18",
+    "express": "^4.18.2",
+    "mongoose": "^7.6.0",
     "mongodb": "6.1",
```

Comment:
this npm package is redundant as using mongoose

Reply:
Done